src/Runner.Common/RunnerDotcomServer.cs

```diff
@@ -46,7 +46,11 @@ public async Task<List<TaskAgent>> GetRunnerByNameAsync(string githubUrl, string
             var githubApiUrl = "";
             var gitHubUrlBuilder = new UriBuilder(githubUrl);
             var path = gitHubUrlBuilder.Path.Split('/', '\\', StringSplitOptions.RemoveEmptyEntries);
-            if (path.Length == 1)
+            var isOrgRunner = path.Length == 1;
+            var isRepoOrEnterpriseRunner = path.Length == 2;
+            var isRepoRunner = !string.Equals(path[0], "enterprises", StringComparison.OrdinalIgnoreCase);
```

Comment:
Would it be better to say 
```csharp
var isRepoRunner = isRepoOrEnterpriseRunner && !string.Equals(path[0], "enterprises", StringComparison.OrdinalIgnoreCase);
```
Otherwise, `isRepoRunner` would be true even in the org case (may not matter how you're using it, but it'd be good to not have a wrong value if we use it in the future)

Reply:
Updated!